site/sfguides/src/data_mapping_in_native_apps/data_mapping_in_native_apps.md

```diff
@@ -387,28 +401,7 @@ st.button('UPDATE!', on_click=update_table)
 
 ### Finishing the Application
 
-All the pieces are in place for our application so we now need to go back to the provider account and upload the files we have just created (manifest.yml, setup_script.sql and enricher_dash.py) to the stage **APPLICATION_STAGE**.  
-
-
-> aside positive
-> 
-> **Note** Remember our application manifest is expectng the file enricher_dash.py to be in a folder called **ui**
-
-once uploaded the stage should look like the following
-
-<img src="assets/code_complete.png" width="395" />
-
-Once the files are uploaded, go into your worksheets and finish off the building of the application package:
-
-```sql
-ALTER APPLICATION PACKAGE IP2LOCATION_APP
-  ADD VERSION MyFirstVersion
-  USING '@IP2LOCATION.IP2LOCATION.APPLICATION_STAGE';
-
-ALTER APPLICATION PACKAGE IP2LOCATION_APP SET DEFAULT RELEASE DIRECTIVE VERSION = MyFirstVersion PATCH = 0;
-```
-
-Our application package is built, now let's create an application.
+All the pieces are in place for our application, we just created (manifest.yml, setup_script.sql and enricher_dash.py) and now we are going to actually deploy it in the next page, in a manual environment you would have to upload the files to the stage APPLICATION_STAGE and then run other commands from a SQL worksheet, but we are not covering that.  
```

Comment:
```suggestion
All the pieces are in place for our application, we just created (manifest.yml, setup_script.sql and enricher_dash.py) and now we are going to actually deploy it in the next step.  
```

I suggest that we mention the manual alternative in the next page, where we actually show the deployment.